### ISSUE TITLE
how to generate strings from multiple .strings file

### ISSUE THREAD

**Issue by embassem** [SwiftGen/SwiftGen#497]
how to generate  all the strings in multiple Strings file . 
i had many *.strings file 
like  
$(SRROOT)/Resources/Localize/Localizable.strings
$(SRROOT)/Resources/Localize/Auth.strings
$(SRROOT)/Resources/Localize/News.strings
is there a way to give  a Path  not a  file in the  .swiftgen.yml  instead of file path 
``` YML
 strings: 
  paths: Resources/Locale/Base.lproj/Localizable.strings 
  templateName: structured-swift4 
  output: Resources/Locale/Localized.swift 
  fallback: Base```

and  a lot of .string file may be added later , so it would be great  to not explicate add each file

**Comment by AliSoftware** [comment]
The key `paths` in your configuration file can be an array of files instead of a file. So you can do this at least:

```yaml
input_dir: Resources/Locale/Base.lproj/
strings:
  paths:
    - Localizable.strings
    - Auth.strings
    - News.strings
  templateName: structured-swift4 
  output: Resources/Locale/Localized.swift 
```

We have an issue open (https://github.com/SwiftGen/SwiftGen/issues/355) to allow glob patterns in the future though, so you'd be able both to use env vars (`$SRCROOT`) and to use `Resources/Localize/*.strings` for example. That hasn't been implemented yet, and isn't planned for the soon-to-be-released 6.0, but any PR to implement that #355 would be welcome anyway :)

**Comment by AliSoftware** [comment]
Btw just saw that we already have an issue to track this feature we want to add in the future: https://github.com/SwiftGen/SwiftGen/issues/32.

Closing this one to avoid duplicates, but don't hesitate to comment/reply on #32 to continue the conversation.
